### DISCUSSION TITLE
关于插件的建议

### DISCUSSION THREAD

**Discussion by PD1014** [MuiseDestiny/zotero-style#458] [Ideas]
你好，博主。我有一个建议，不知道它的实现难度大不大，想给您提一下。
在我最近的使用中，如果有两台电脑在使用zotero-style  通过网络同步是没有办法  同步其在一台电脑上的设置的，博主能否设置一个输出、输入配置的接口，通过输出、输入配置的形式实现，zotero-style 在各种设置上的同步。

**Comment by MuiseDestiny** [comment]
这个在todo中，会做的。
